src/test/java/com/jcabi/aether/MavenClasspathTest.java

```diff
@@ -55,37 +56,79 @@
 public final class MavenClasspathTest {
 
     /**
+     * Sdirectory format.
+     */
+    private static final String SDIR = "%sas%<sdirectory";
+
+    /**
+     * File separator.
+     */
+    private static final String FILE_SEP = "file.separator";
+
+    /**
      * Temp dir.
      * @checkstyle VisibilityModifier (3 lines)
      */
     @Rule
     public final transient TemporaryFolder temp = new TemporaryFolder();
 
     /**
-     * Classpath can build a classpath.
+     * MavenClasspath can build a classpath.
      * @throws Exception If there is some problem inside
      */
     @Test
     public void buildsClasspath() throws Exception {
-        final Dependency dep = new Dependency();
         final String group = "junit";
-        dep.setGroupId(group);
-        dep.setArtifactId(group);
-        dep.setVersion("4.10");
-        dep.setScope("test");
         final String jar = "junit-4.10.jar";
         final DependencyGraphBuilder builder = this.builder(jar);
         final MavenSession session = Mockito.mock(MavenSession.class);
-        final MavenProject project = this.project(dep);
+        final MavenProject project = this.project(
+            this.dependency(
+                group, group, "4.10"
+            )
+        );
+        Mockito.when(session.getCurrentProject()).thenReturn(project);
+        MatcherAssert.assertThat(
+            new MavenClasspath(builder, session, JavaScopes.TEST),
+            Matchers.<File>hasItems(
+                Matchers.hasToString(
+                    Matchers.endsWith(
+                        String.format(
+                            MavenClasspathTest.SDIR,
+                            System.getProperty(MavenClasspathTest.FILE_SEP)
+                        )
+                    )
+                ),
+                Matchers.hasToString(Matchers.endsWith(jar))
+            )
+        );
+    }
+
+    /**
+     * MavenClasspath can build a classpath when there are more scopes.
+     * @throws Exception If there is some problem inside
+     */
+    @Test
+    public void buildsClasspathWithMoreScopes() throws Exception {
+        final String jar = "jcabi-xml-0.17.2.jar";
+        final DependencyGraphBuilder builder = this.builder(jar);
+        final MavenSession session = Mockito.mock(MavenSession.class);
+        final MavenProject project = this.project(
+            this.dependency(
+                "com.jcabi", "jcabi-xml", "0.17.2"
```

Comment:
@mkordas I understand the variable reusing principle,  but I really think it is not necessary because they are not the same, strings. It's not "0.17.2" as is, used in 2 places, but compounds of it. Do you really think it's necessary to start composing strings just for this?? I think it will make the test more hard to read...

Again, not that I don't want to do it - it took me more to write this message. I just don't think it's ok. Do you really think so?